src/wmts/capabilities.ts

```diff
@@ -171,6 +171,7 @@ export function readLayersFromCapabilities(
       const style: LayerStyle = {
         title: getElementText(findChildElement(element, 'Title')),
         name: getElementText(findChildElement(element, 'Identifier')),
+        abstract: getElementText(findChildElement(element, 'Abstract')),
```

Comment:
IIRC this is an optional property of the style; do you think you could make it so that the property is only defined if the abstract is present? like it is done for `legendUrl`. Thanks!

Reply:
Thanks for the feedback, hopefully that should be sorted now!

Reply:
this looks like it is done for WMS, could you do the same for WMTS? thanks :)

Reply:
Oof, glad someone's awake, here's hoping that's sorted now.